### ISSUE TITLE
Serious problem with bluetooth keyboard & duscussions.apple.com

### ISSUE THREAD

**Issue by SeAlgoAsoma** [brave/browser-ios#223]
I use a Zagg BT keyboard with my iPad, and I was just over in Apple's forums, and while typing a comment in their fancy comment entry gizmo, it mysteriously stopped accepting input from the keyboard. I tried a few more times on a few more pages on the same site, and it happened pretty reliably... Making using that site almost impossible. I tried Safari, and had no problem like this. 

This is a dealkiller for me. I absolutely need to be able to type comments on that site without having to futz around reloading pages and clicking around trying to get the form box to be 'alive' again, if there isn't a simple fix I can't use this browser.

**Comment by garvankeeley** [comment]
This is an odd bug. The web view itself is an apple-provided component, so we don't control the hardware keyboard interaction with the web view. I can't think of anything in the code that would interfere with this, hopefully that is the case so I can fix this. I have an old BT keyboard I'll dig up and try this and see.

**Comment by SeAlgoAsoma** [comment]
Yeah, it seems odd, but I've noticed this with a few third-party browsers and my Zagg folio BT keyboard... Dolphin was my browser of choice until these sorts of things drove me away from it. They don't seem to happen in Safari. 

Could it be a function of... What's it called... Doesn't mobile safari use a special javascript engine that's not available to 3rd party browsers? "Nitro" I think?
